### PR TITLE
live-images: remove outdated flavor image info and simplify

### PR DESCRIPTION
**Pull request by bobertlo**

### CODE REVIEW

File: src/installation/live-images/index.md [hunk 1]
```diff
@@ -1,55 +1,22 @@
...
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64`, and for `glibc` only on `i686`. Users of other
+architectures will need to perform a [manual installation](../guides/index.md).
+
+## Base Image
+
+The base images contain just enough for a usable Void system, with utilities for
+basic hardware and network configuration included. Packages can be installed
```
Comment (paper42) [inline src/installation/live-images/index.md:12]:
```suggestion
The base image contains just enough packages to install a usable Void system, with utilities for
basic hardware and network configuration included. Packages can be installed
```

the `with...` sounds a bit weird to me

File: src/installation/live-images/index.md [hunk 2]
```diff
@@ -1,55 +1,22 @@
...
+from the image offline, or by downloading the latest versions online.
+
+## XFCE Desktop Image
+
+Void also offers an XFCE image. The live environment and installed system
+include a desktop environment, web browser, and basic applications
+pre-configured, but the installation procedure is the same. Void must be
```
Comment (paper42) [inline src/installation/live-images/index.md:19]:
```suggestion
Void also offers an XFCE image. The live environment
includes a desktop environment, web browser and basic desktop applications,
but the installation procedure is the same. Void must be
```

**Review by paper42 — COMMENTED** [review]
I think we should also mention other DEs are also supported, I heard people saying that void only supports XFCE because it's minimal and doesn't "depend on systemd".

File: src/installation/live-images/index.md [hunk 3]
```diff
@@ -1,55 +1,22 @@
...
-| Other             | Mixer, EConnMan (connection manager), Elementary Test | -               | LXTask (task manager), MIME type editor | Screen grabber | Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer | Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder |
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64`, and for `glibc` only on `i686`. Users of other
```
Comment (ericonr) [inline src/installation/live-images/index.md:6]:
```suggestion
on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
```

File: src/installation/live-images/index.md [hunk 4]
```diff
@@ -1,55 +1,22 @@
...
-| Archive unpacker  | -                                                     | -               | -                                       | -              | Engrampa                                                                                                                                                            | -                                                                                                                                   |
-| Other             | Mixer, EConnMan (connection manager), Elementary Test | -               | LXTask (task manager), MIME type editor | Screen grabber | Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer | Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder |
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
```
Comment (ericonr) [inline src/installation/live-images/index.md:4]: "in the mirrors", I think?

File: src/installation/live-images/index.md [hunk 5]
```diff
@@ -1,55 +1,22 @@
...
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64`, and for `glibc` only on `i686`. Users of other
+architectures will need to perform a [manual installation](../guides/index.md).
+
+## Base Image
+
+The base images contain just enough for a usable Void system, with utilities for
+basic hardware and network configuration included. Packages can be installed
+from the image offline, or by downloading the latest versions online.
```
Comment (ericonr) [inline src/installation/live-images/index.md:13]: This isn't clear that it means installing packages onto a new install.

Comment (ericonr) [reply src/installation/live-images/index.md:13]: Oh, I see paper had noticed this already!

File: src/installation/live-images/index.md (hunk 4 above)
Comment (bobertlo) [reply src/installation/live-images/index.md:4]: I am not an English major, but I think on is appropriate here?

File: src/installation/live-images/index.md [hunk 6]
```diff
@@ -1,55 +1,24 @@
...
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
+architectures will need to perform a [manual installation](../guides/index.md).
+
+## Base Image
+
+The base image contains just enough for a usable Void system, with utilities for
+basic hardware and network configuration included. The installation can be
+performed online, which will download the latest package versions versions, or
```
Comment (paper42) [inline src/installation/live-images/index.md:13]:
```suggestion
performed online, which will download the latest package versions, or
```

File: src/installation/live-images/index.md [hunk 7]
```diff
@@ -1,55 +1,24 @@
...
-| Other             | Mixer, EConnMan (connection manager), Elementary Test | -               | LXTask (task manager), MIME type editor | Screen grabber | Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer | Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder |
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
+architectures will need to perform a [manual installation](../guides/index.md).
+
+## Base Image
+
+The base image contains just enough for a usable Void system, with utilities for
```
Comment (paper42) [inline src/installation/live-images/index.md:11]:
```suggestion
The base image contains just enough tools for installing a usable Void system, with utilities for
```

File: src/installation/live-images/index.md [hunk 8]
```diff
@@ -1,55 +1,24 @@
...
+basic hardware and network configuration included. The installation can be
+performed online, which will download the latest package versions versions, or
+offline, which will use the versions of packages available on the installation
+medium.
+
+## XFCE Desktop Image
+
+Void also offers an XFCE image. The live environment includes a desktop
+environment, web browser, and basic desktop applications, but the installation
```
Comment (paper42) [inline src/installation/live-images/index.md:20]:
```suggestion
environment, web browser and basic desktop applications, but the installation
```
at least I think

File: src/installation/live-images/index.md [hunk 9]
```diff
@@ -1,55 +1,24 @@
...
+procedure is the same. Choose the `Local` package source in void-installer to
```
Comment (paper42) [inline src/installation/live-images/index.md:21]:
```suggestion
procedure is the same as with the base image. Choose the `Local` package source in void-installer to
```

File: src/installation/live-images/index.md [hunk 10]
```diff
@@ -1,55 +1,24 @@
...
-| Other             | Mixer, EConnMan (connection manager), Elementary Test | -               | LXTask (task manager), MIME type editor | Screen grabber | Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer | Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder |
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
```
Comment (paper42) [inline src/installation/live-images/index.md:6]:
```suggestion
[mirrors](../../xbps/repositories/mirrors/index.md) with either `glibc` or `musl`
on `x86_64` systems and only with `glibc` on `i686` ones. Users of other
```

File: src/installation/live-images/index.md (hunk 7 above)
Comment (bobertlo) [reply src/installation/live-images/index.md:11]: I think "includes just enough tools to install a usable Void system" works even better?

File: src/installation/live-images/index.md (hunk 8 above)
Comment (bobertlo) [reply src/installation/live-images/index.md:20]: Is Oxford comma against the style guide?

**Comment by bobertlo** [comment]
Someone else got something merged

**Pull request closed without merging**